pandas/tests/io/json/test_pandas.py

```diff
@@ -1662,3 +1664,13 @@ def test_json_multiindex(self, dataframe, expected):
         series = dataframe.stack()
         result = series.to_json(orient="index")
         assert result == expected
+
+    @mock_s3
```

Comment:
I think you'll likely want to use our `s3_resource` fixture to get this to work correctly:
https://github.com/pandas-dev/pandas/blob/fc813e76e7d8863410aa4b1a581dfa777281271c/pandas/tests/io/conftest.py#L29

See `test_read_s3_jsonl` for an example of its usage:
https://github.com/pandas-dev/pandas/blob/fc813e76e7d8863410aa4b1a581dfa777281271c/pandas/tests/io/json/test_pandas.py#L1210-L1216

Might need to make some modifications for write instead of read but should be a good starting point.